src/pybind/mgr/dashboard/services/cluster_service.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import enum
```

Comment:
@epuertat let's have a convention consensus here and state it clearly in the devel docs so we have consistency.

Reply:
@epuertat @alfonsomthd It makes sense to go with `services/cluster.py` rather than cluster_service.py, so I'll make the change here. We can include this in the docs and do a refactor about this later on?

Reply:
Done.